examples/todoapp/src/generatedNoCheck/FoundryClient.ts

```diff
@@ -0,0 +1,13 @@
+import type { Auth, FoundryClientOptions } from '@osdk/legacy-client';
```

Comment:
Do we really want the directory to be called `generatedNoCheck`?

Reply:
the eslint rules were shortcutting the `generatedNoCheck` for the other location we generate code so i just kept the convention.